dev/TabView/TabViewItem.h

```diff
@@ -72,4 +72,7 @@ class TabViewItem :
 
     void UpdateShadow();
     winrt::IInspectable m_shadow{ nullptr };
+
+    winrt::IInspectable TabViewItem::ResourceLookup(const winrt::Control& control, const winrt::IInspectable& key);
```

Comment:
replace or add with the one in SharedHelpers

Reply:
Since there wasn't a helper for that, I've added the function to ResourceAccessor. Since AutoSuggestBoxHelper and ComboBoxHelper use the exact same function, I've refactored them to now use the helper too, to remove duplicate code.